lib/extensions/descendant_loader.rb

```diff
@@ -261,8 +281,9 @@ def scoped_name(name, scopes)
   end
 
   module ArDescendantsWithLoader
+    # reload_schema_from_cache __update_callbacks descendants subclasses
```

Comment:
Remove this commit.  Provide a brief example of where these methods calls are coming from so it's clear why descendants and subclasses have different lists of caller methods.